docs/8-veba-troubleshooting.md

```diff
@@ -0,0 +1,232 @@
+# vCenter Event Broker Appliance - Troubleshooting
+
+## Requirements
+
+You must log on to the VEBA appliance as root. You can do this from the console. If you want SSH access, execute the following command:
+
+```
+systemctl start sshd 
+```
+
+This turns on the SSH daemon but does not enable it to start on appliance boot. You should now be able to SSH into the VEBA appliance. 
```

Comment:
I think we should not use the unofficial VEBA acronym. To avoid repetition of the long name, perhaps writing about the "appliance" is sufficient.

Reply:
Changed initial reference to full name, all other references to 'appliance' or 'the appliance'